### ISSUE TITLE
Error when downloading from provider Legendas.TV

### ISSUE THREAD

**Issue by jotavip** [morpheus65535/bazarr#1584]
When using the Brazilian provider legendasTV on Bazarr to search subtitles, it is returning the following error: 

Throttling legendastv for 10 minutes, until 21/10/17 22:42, because of: ProviderError. Exception info: 'Archive timestamp is in the future'

The time of the log is incorrect, it is about three hours late than my QNAP system time. I think this is the issue. How can I fix this?

**Software (please complete the following information):**
Bazarr Version
1.0.0
Sonarr Version
3.0.6.1342
Radarr Version
3.2.2.5080
Operating System
Linux-3.4.6-x86_64-with-glibc2.2.5
Python Version
3.8.1

**Comment by morpheus65535** [comment]
You probably haven't properly set your time zone settings on your QNAP. Anything different from your previous issue? https://github.com/morpheus65535/bazarr/issues/1436

**Comment by jotavip** [comment]
Hi, I checked the QNAP time zone and it is correct. I attached the screenshot of the settings. After some time, Bazarr downloaded the subtitles that I had this issue, I think because it reached the timestamp. 

The previous issue was solved, it downloaded the subtitles. But if it is a new subtitle on the provider, it gives this time stamp issue
![2021-10-19](https://user-images.githubusercontent.com/4889022/137840511-1d67e9b0-9478-4485-9dce-b97b4749fcb2.png)
.
